src/lib/components/Section.svelte

```diff
@@ -62,7 +62,6 @@ Layout utility component for displaying a major site section with grid-based con
 <style lang="postcss">
 	.section {
 		padding: var(--section-padding-y, 0) var(--section-padding-x, 0);
-		overflow: auto;
```

Comment:
@allozaur note that removing this causes some layouts to break (e.g., `/glossary`). I fixed this for now by manually adding `overflow: auto` to the relevant section. But this is why I think we need a more universal solution to `sticky` that doesn't conflict with `overflow: auto`.